Client/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -115,7 +183,7 @@ class SyncSettingsTableViewController: UITableViewController {
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         let deviceCount = frc?.fetchedObjects?.count ?? 0
         
-        return section == Sections.buttons.rawValue ? ButtonsSection.count : deviceCount
+        return section == Sections.buttons.rawValue ? 1 : deviceCount
```

Comment:
Can you add a comment on this?

Reply:
`remove device` button is gone, and we are left with one button so I decided to drop the buttons `enum` altogether.